pyvista/plotting/qt_plotting.py

```diff
@@ -323,6 +327,11 @@ def __init__(self, parent=None, title=None, shape=(1, 1), off_screen=None,
 
         self.signal_set_view_vector.connect(self.view_vector)
         self.signal_reset_camera.connect(self.reset_camera)
+        # self.render_trigger.connect(self._render)
+        self.signal_enable_trackball_style.connect(self.enable_trackball_style)
+        self.signal_remove_legend.connect(self.remove_legend)
+        self.signal_set_background.connect(self.set_background)
+        self.signal_remove_actor.connect(self.remove_actor)
```

Comment:
I see those signals are created and connected but when are they emitted?

Reply:
Ooohhh. Okay I see now

I’m trying to teach myself proper PyQt development and these signals were confusing me. I’ll override those methods to emit a signal on the superclass

Reply:
Can anyone comment on when these signals are needed vs when they are not? Why do we need a signal for `set_background_color` and not for other things like setting the camera position?

Should we set up these signals for all methods?